assets/dev/js/editor/views/add-section/base.js

```diff
@@ -185,6 +187,34 @@ import environment from 'elementor-common/utils/environment';
 		};
 	}
 
+	onGridPresetSelected( event ) {
+		this.closeSelectPresets();
+
+		const selectedStructure = event.currentTarget.dataset.preset;
+		const chunks = selectedStructure.split( '-' );
+		const rows = chunks[ 0 ];
+		const columns = chunks[ 1 ];
```

Comment:
@ArmanPurtoian Is it possible to use `elementor.presetsFactory.getParsedStructure()` to parse the structure?